lib/pbench/test/unit/server/test_schema.py

```diff
@@ -299,6 +299,12 @@ def test_invalid_optional(self, json):
         x = Parameter("data", ParamType.STRING, required=False)
         assert not x.invalid(json)
 
+    @pytest.mark.parametrize("value,expected", (("", True), ("true", True)))
+    def test_boolean(self, value, expected):
```

Comment:
No `("false", False)` case (and no `"yes"`, `"no"`, or single-letter cases?)?

Reply:
I tried to avoid the plethora of test cases hitting the same statement.